src/lib/leagues/hardTasks.ts

```diff
@@ -0,0 +1,1045 @@
+			return Object.keys(mahojiUser.monsterScores as ItemBank).length >= 75;
+		}
+	},
+	{
+		id: 2063,
+		name: 'Slay 50 superior slayer creatures',
+		has: async ({ mahojiUser }) => {
+			return mahojiUser.slayer_superior_count >= 50;
+		}
+	},
+	{
+		id: 2064,
+		name: 'Sacrifice 3b worth of items/GP',
+		has: async ({ mahojiUser }) => {
+			return mahojiUser.sacrificedValue >= 3_000_000_000;
+		}
+	},
+	{
+		id: 2065,
+		name: 'Complete the Corrupted Gauntlet',
+		has: async ({ minigames }) => {
+			return minigames.corrupted_gauntlet >= 1;
+		}
+	},
+	{
+		id: 2066,
+		name: 'Complete the Corrupted Gauntlet 100 times',
+		has: async ({ minigames }) => {
+			return minigames.corrupted_gauntlet >= 100;
+		}
+	},
+	{
+		id: 2067,
+		name: 'Unlock every invention blueprint',
+		has: async ({ mahojiUser }) => {
+			return mahojiUser.unlocked_blueprints.length >= Inventions.length;
+		}
+	},
+	{
+		id: 2068,
+		name: 'Dye an item',
+		has: async ({ cl }) => {
+			return dyedItems
+				.map(i => i.dyedVersions)
+				.flat(2)
+				.some(i => cl.has(i.item.id));
+		}
+	},
+	{
+		id: 2069,
+		name: 'Achieve level 120 Agility',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.agility >= 120;
+		}
+	},
+	{
+		id: 2070,
+		name: 'Achieve level 120 Cooking',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.cooking >= 120;
+		}
+	},
+	{
+		id: 2071,
+		name: 'Achieve level 120 Fishing',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.fishing >= 120;
+		}
+	},
+	{
+		id: 2072,
+		name: 'Achieve level 120 Mining',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.mining >= 120;
+		}
+	},
+	{
+		id: 2073,
+		name: 'Achieve level 120 Smithing',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.smithing >= 120;
+		}
+	},
+	{
+		id: 2074,
+		name: 'Achieve level 120 Woodcutting',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.woodcutting >= 120;
+		}
+	},
+	{
+		id: 2075,
+		name: 'Achieve level 120 Firemaking',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.firemaking >= 120;
+		}
+	},
+	{
+		id: 2076,
+		name: 'Achieve level 120 Runecraft',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.runecraft >= 120;
+		}
+	},
+	{
+		id: 2077,
+		name: 'Achieve level 120 Crafting',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.crafting >= 120;
+		}
+	},
+	{
+		id: 2078,
+		name: 'Achieve level 120 Prayer',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.prayer >= 120;
+		}
+	},
+	{
+		id: 2079,
+		name: 'Achieve level 120 Fleching',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.fletching >= 120;
+		}
+	},
+	{
+		id: 2080,
+		name: 'Achieve level 120 Farming',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.farming >= 120;
+		}
+	},
+	{
+		id: 2081,
+		name: 'Achieve level 120 Herblore',
+		has: async ({ skillsLevels }) => {
+			return skillsLevels.herblore >= 120;
+		}
+	},
+	{
+		id: 2082,
```

Comment:
This is a duplicate, there are 2x Herblore here.

I checked, there are 26 here and only 25 skills, so nothing is missing, herblore is just doubled.